digdag-tests/src/test/resources/acceptance/group_retry/retry-1.dig

```diff
@@ -0,0 +1,10 @@
++step1:
+  sh>: touch ${outdir}/${attempt_id}1-1.out
++step2:
+  _retry: 2
```

Comment:
All the group tasks in this test have ` _retry: 2`, but with https://github.com/treasure-data/digdag/pull/1419/files#diff-55f697f2343f66f95a3fd8c57c0710a6R309 the existence of ` _retry: 2` switches `retry --resume`'s behaviour. How about adding tests that don't contain `_retry: 2`?